client/templates/users/userEdit.js

```diff
@@ -1,9 +1,56 @@
 Template.userEdit.helpers({
-  //add you helpers here
+  schema: function () {
+    var classSchema = new SimpleSchema({
+      classId: {
+        type: String,
+        allowedValues: function () {
```

Comment:
@ajax-wcd it would have to stay here regardless. The reason being that, since the form doesn't use the regular schema, it needs to have allowedValues in the form schema to validate the data client-side before submission to the server method.